ci-operator/jobs/openshift/openshift-azure/openshift-openshift-azure-periodics.yaml

```
@@ -23,7 +23,7 @@ periodics:
         valueFrom:
           configMapKeyRef:
             key: openshift-openshift-ansible-release-3.11.yaml
-            name: ci-operator-3.x-configs
+            name: ci-operator-misc-configs
```

And these are periodics that reference 3.11 configs of `openshift/openshift-ansible`...

```suggestion
            name: ci-operator-3.x-configs
```

---

for fuck's sake